core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -272,6 +273,13 @@ public static FileIO loadFileIO(
     configureHadoopConf(fileIO, hadoopConf);
 
     fileIO.initialize(properties);
+
+    // Check if we should wrap this with CachingFileIO
```

Comment:
Minor: personal pronouns like "I" and "we" usually make comments longer and harder to read. More direct statements like `Check whether to wrap with CachingFileIO` are preferred.

Reply:
Done